TrainingExtensions/tensorflow/src/python/aimet_tensorflow/quantsim.py

```diff
@@ -691,6 +701,26 @@ def _insert_param_quantization_ops(self, op_names: List[str], indices: List[int]
                 if nodes_modified_count != 1:
                     raise ValueError('Input ' + param_in.name + ' not quantized!')
 
+    @staticmethod
+    def _get_number_of_output_channels(op: tf.Operation, index: int):
+        """
+        Gets number of output channels and quantization axis for an op for per channel quantization
+        """
+        weight_shape = op.inputs[index].get_shape().as_list()
+        # If weight shape length is 1, the parameter is bias
+        num_output_channels = None
+        axis = None
+        if len(weight_shape) == 1:
+            num_output_channels = weight_shape[0]
+            axis = 0
+        elif op.type in ['Conv2D', 'DepthwiseConv2dNative']:
+            num_output_channels = weight_shape[3]
+            axis = 3
```

Comment:
Is this actually correct? I thought that TF supported different memory layouts. like NCHW and NHWC